src/argilla/client/feedback/training/schemas.py

```diff
@@ -131,16 +135,17 @@ def for_text_classification(
             LabelQuestionUnification,
             MultiLabelQuestionUnification,
             RankingQuestionUnification,
-        ],
+        ] = None,
         label_strategy: str = None,
     ) -> "TrainingTaskForTextClassification":
         """
-        _summary_
+        Define a task configuration for text classification. It takes default values for `text` and `label` using datasets Fields and Questions or a custom `formatting_func` as Callable. See Examples underneath for more details.
 
         Args:
-            text (TextField): The TextField to use for training.
-            label (Union[RatingQuestion, LabelQuestion, RankingQuestion, MultiLabelQuestion, RatingQuestionUnification, LabelQuestionUnification, MultiLabelQuestionUnification, RankingQuestionUnification]): _description_
-            label_strategy (str, optional): A strategy to unify responses. Defaults to None. This means it will initialize the default strategy for the label type.
+            formatting_func (Callable[[Dict[str, Any]], Union[None, str, List[str], Iterator[str]]], optional): A formatting function. Defaults to None.
+            text (TextField, optional): The TextField to use for training. Defaults to None.
+            label (Union[RatingQuestion, LabelQuestion, RankingQuestion, MultiLabelQuestion, RatingQuestionUnification, LabelQuestionUnification, MultiLabelQuestionUnification, RankingQuestionUnification], optional): The *Question to use for training. Defaults to None.
+            label_strategy (str, optional): A strategy to unify responses. Defaults to None. This means it will initialize the default strategy for the label type. Defaults to None.
```

Comment:
```suggestion
            label_strategy: A strategy to unify responses. Defaults to None. This means it will initialize the default strategy for the label type. Defaults to None.
```